lib/browser/commands/assert-view/index.js

```diff
@@ -44,7 +44,13 @@ module.exports = (browser) => {
         }
 
         const {canHaveCaret, pixelRatio} = page;
-        const imageCompareOpts = {tolerance, antialiasingTolerance, canHaveCaret, pixelRatio, compareOpts};
+        const imageCompareOpts = {
+            tolerance: typeof opts.tolerance === 'number' ? opts.tolerance : tolerance,
```

Comment:
It would be better to set default `opts.tolerance` in line 20. Something like:
```js
opts = _.defaults(opts, {ignoreElements: [], tolerance});
...
const imageCompareOpts = {
    tolerance: opts.tolerance
...
```